lib/wpscan/db/updater.rb

```diff
@@ -24,6 +24,7 @@ def initialize(repo_directory)
 
         FileUtils.mkdir_p(repo_directory.to_s) unless Dir.exist?(repo_directory.to_s)
 
+        return if ParsedCli.update == "false"
```

Comment:
I think this should be a `false` literal, not a string:
```suggestion
        return if ParsedCli.update == false
```